api/k8s/application/k8sApplicationRestHandler.go

```diff
@@ -204,10 +240,52 @@ func (handler *K8sApplicationRestHandlerImpl) GetResource(w http.ResponseWriter,
 		//setting devtronAppIdentifier value in request
 		request.DevtronAppIdentifier = devtronAppIdentifier
 		request.ClusterId = request.DevtronAppIdentifier.ClusterId
+		appId := request.DevtronAppIdentifier.AppId
+		envId := request.DevtronAppIdentifier.EnvId
+		cdPipeline, err := handler.k8sApplicationService.GetPipelineByAppIdEnvId(appId, envId)
```

Comment:
why is this method in k8's application service. this logic should be in pipeline service or pipeline config service.